lib/game.rb

```diff
@@ -0,0 +1,102 @@
+require_relative './player.rb'
+require_relative './board.rb'
+
+class Game
+	attr_accessor :board, :player_1, :player_2, :current_player
+
+	WIN_COMBINATIONS = [
+	[0, 1, 2],
+	[3, 4, 5],
+	[6, 7, 8],
+	[0, 3, 6],
+	[1, 4, 7],
+	[2, 5, 8],
+	[0, 4, 8],
+	[2, 4, 6]
+	]
+
+	@@wins = 0
+	@@draws = 0
+	def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+		@player_1 = player_1
+		@player_2 = player_2
+		@board = board
+		@current_player = player_1
+		@winning_symbol
+	end
+
+	def over? 
+		self.won? || self.draw? ? true : false
+	end
+
+	def won?
+		WIN_COMBINATIONS.each do |combo|
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```